samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/SectionActivityListener.java

```diff
@@ -171,15 +171,11 @@ private Map getUserIdNameMap(List list) {
         while (it.hasNext()) {
             EnrollmentRecord enr = (EnrollmentRecord) it.next();
             String uid = enr.getUser().getUserUid();
-            String firstName ="";
-            String lastName="";
             String displayName="";
             try {
-                firstName = UserDirectoryService.getUser(uid).getFirstName();
-                lastName = UserDirectoryService.getUser(uid).getLastName();
-                displayName = lastName + ", " + firstName;
+                displayName = enr.getUser().getDisplayName();
             }catch (Exception e) {
-                e.printStackTrace();
+                log.error("Error in getUserIdNameMap", e);
```

Comment:
Hi Jose,
The method will be in the stack trace so an error message like "Error in SomeMethod" is redundant!

Since UserDirectoryService.getUser() is removed the try catch is probably not needed!